target/linux/ramips/image/mt7620.mk

```diff
@@ -73,6 +73,31 @@ define Device/amit_jboot
   DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-usb-ohci
 endef
 
+define Device/apexera-netbot
```

Comment:
"apexera_netbot" instead of "apexera-netbot".
Did you build-test, because that should have caused an error?

Reply:
I figured out I have tested the commit maybe one or two week ago before we have made the new rule for device image building and dts binding with it.

Reply:
For now, we don't have to define the DTS variable with image building, and the dts file name is defined with the target name, right?

Reply:
The DTS file name is constructed from MTK_SOC and the device definition name:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/image/Makefile#L28